src/sssom_schema/schema/sssom_schema.yaml

```
@@ -557,6 +558,20 @@ slots:
         description: A URL pointing to the AgreementMakerLight project.
       - value: AgreementMakerLight
         description: A string (name) denoting the AgreementMakerLight project.
+  mapping_tool_id:
+    description: The ID (entity reference) of the tool or algorithm that was used to generate the
+      mapping.
+    range: EntityReference
+    instantiates: sssom:Propagatable
```

```suggestion
    instantiates:
      - sssom:Propagatable
      - sssom:Versionable
```

You need to instantiates `sssom:Versionable` as well in order to use the `added_in` annotation.